src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const clone = {};
+
+  for (const key in robot) {
+    const newKey = robot[key];
+
+    if (clone.hasOwnProperty(newKey)) {
+      return null;
+    }
+    clone[newKey] = key;
```

Comment:
It's good practice to check if the property belongs to the object itself and is not inherited from the prototype chain. You can add a check using 'robot.hasOwnProperty(key)' inside the loop.